concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -0,0 +1,345 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
```

Comment:
Does this list come with any attributes with respect to sound symbolism in the original data? If so, they should be added. Should I have a look? If so, please remind me explicitly, e.g., also by writing an email to me.

Reply:
The data for 125 concepts out of 344 can be found here: https://osf.io/yzvq9/. Shall I then add all the columns including the  one with the sound groups?

Reply:
@LinguList could you have a look at the data following the link above? Each of the 125 concepts has a different number of entries for different sound patterns. Which columns should be added to Concepticon then and in which way? Do we also need the transcriptions?